### PR TITLE
Azc module

### PR DESCRIPTION
**Pull request by dionytadema**
the bot now will try to build an azc when it is not already present

### CODE REVIEW

File: Municipality/azc.mod2g [hunk 1]
```diff
@@ -0,0 +1,16 @@
+use dummy as knowledge.
+
+exit=nogoals.
+
+order = linearall.
+module azc{
+	%%% check if there is still need for an azc
+	if not(bel(building(Id,_,OwnerID,_,_,650,_))) then adopt(buildAZC).
```
Comment (levilime) [inline Municipality/azc.mod2g]: Please document the meaning of numbers

File: Municipality/azc.mod2g [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	%%% check if we have budget for an azc
+	%minstens 15M
```
Comment (levilime) [inline Municipality/azc.mod2g]: This is dutch, please write all code in english

**Comment by StefanV7** [comment]
Looks good :+1:

**Comment by levilime** [comment]
Could you add a rudimentary test to verify functionality (also for integration purposes)

**Comment by dionytadema** [comment]
tests in the current version of goal are broken, even changing the tests nick wrote will not make the test fail

**Comment by NickTUD** [comment]
- Fix conflicts
- For next commits, please submit more descriptive messages...

File: Municipality/test.test2g [hunk 3]
```diff
@@ -25,13 +25,25 @@ test tygron with
...
-		
 	}
-	post{true}
+	post{true}		
+	
+test azc with
+	pre{true}
+	in{ 
+		%Test azc module (azc has id 650)
+		percept(money(1)), not(bel(building(ID,Name,OwnerID,ConstructionYear,CategoriesList,650,Floors)))
```
Comment (NickTUD) [inline Municipality/test.test2g:47]: Change ID to BuildingID

**Comment by StefanV7** [comment]
I think ID in building is intuitive enough for people to understand it refers to the ID of the building, which is the real issue.

File: Municipality/azc.mod2g [hunk 4]
```diff
@@ -0,0 +1,13 @@
+use dummy as knowledge.
+
+order = linearall.
+module azc{
+	%%% check if there is still need for an azc (id 650)
+	if bel(building(Id,_,OwnerID,_,_,650,_)) then exit-module.
+	%%% check if we have budget for an azc
+	if bel(money(X),X<100000) then exit-module.
```
Comment (NickTUD) [inline Municipality/azc.mod2g:8]: Let comments start with a capital letter for consistency (one of the current issues from our Code Evolution Quality)

**Comment by dionytadema** [comment]
could someone else look into this conflict, i can't seem to fix it

**Comment by StefanV7** [comment]
Looks good still :+1:

**Pull request merged**